src/view/domconverter.js

```diff
@@ -278,6 +278,12 @@ export default class DomConverter {
 				domAfter = domParent.childNodes[ 0 ];
 			} else {
 				domBefore = this.getCorrespondingDom( viewPosition.nodeBefore );
+
+				if ( !domBefore ) {
```

Comment:
I guess you tried to fix cases like https://github.com/ckeditor/ckeditor5-autoformat/issues/13#issuecomment-259352223. I'm generally ok with this, because the only alternative is that `viewPositionToDom` will be called in a `try-catch` which smells really bad. So if we agree that `viewPositionToDom` may return `null`, then we should mention it in the documentation.

But then we should perhaps handle this more holistically in this method. E.g. `getCorrespondingDom()` may return `null` also in other cases here, so those would need to be handled as well.

cc @pjasiun